account_analytic_tag/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+Analytical tags.
```

Comment:
Please elaborate this a bit more, and put in `CONFIG.rst` the part that belongs there.